win32ss/gdi/ntgdi/bitmaps.c

```diff
@@ -333,6 +333,11 @@ IntCreateCompatibleBitmap(
                           Planes ? Planes : 1,
                           Bpp ? Bpp : dibs.dsBm.bmBitsPixel,
                           NULL);
+            if (Bmp == NULL)
+            {
+                DPRINT1("Failed to allocate a bitmap!\n");
+                return NULL;
+            }
```

Comment:
Interesting that in line 350 below, this is `psurf` that is acted upon (for setting `SurfObj.hdev`) and not `psurfBmp`. Is there an explanation for this?

Reply:
Yeah, that looks like copypasta. And it's there since 2014.